velox/dwio/dwrf/test/ConfigTests.cpp

```diff
@@ -71,6 +78,79 @@ struct ConfigTestParams {
   std::vector<uint32_t> expectedCols{}; // do we expect the spec to be valid
 };
 
+TEST(ConfigTests, WriterOptionsDefaultConfig) {
+  WriterOptions options;
+  const facebook::velox::config::ConfigBase base({});
+  const facebook::velox::config::ConfigBase emptySession({});
+
+  options.processConfigs(base, emptySession);
+  auto config = options.config;
+  ASSERT_EQ(getConfig(config, Config::STRIPE_SIZE), 64L * 1024L * 1024L);
+  ASSERT_EQ(
```

Comment:
I suppose here we are copying the exact same tests from the hive connector? My only concern is that we don't change the default values (as these are currently all over the place), since this may have hard to predict effects. But I'm assuming all of these are preserved.

Reply:
The API changed a bit since we moved from HiveConfig to Dwrf::Config and `processConfigs`. But the body and default values are the same.
There are 8 configs and I confirmed the defaults are the same as previous.